katharsis-core/src/main/java/io/katharsis/resource/registry/repository/adapter/ResponseRepositoryAdapter.java

```diff
@@ -0,0 +1,348 @@
+package io.katharsis.resource.registry.repository.adapter;
+
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import io.katharsis.module.ModuleRegistry;
+import io.katharsis.queryParams.QueryParams;
+import io.katharsis.queryspec.DefaultQuerySpecConverter;
+import io.katharsis.queryspec.QuerySpec;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.katharsis.queryspec.QuerySpec'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)